pluto_build/data/pluto_input_research.csv

```diff
@@ -27192,4 +27192,6 @@ bbl,field,old_value,new_value,Type,reason,version
 ,ownername,SCHOOL CONSTRUCTION AUTHORITY,NYC SCHOOL CONSTRUCTION AUTHORITY,1,Normalized owner name,22v3
 1011637503,unitsres,21560,196,2,Incorrect increse in number of units taking value from 22v2,22v3
 4024447501,unitsres,4761,69,2,Incorrect increse in number of units taking value from DOB CO,22v3
-1016177502,unitsres,2160,361,2,Incorrect increse in number of units taking value from DOB CO,22v3
\ No newline at end of file
+1016177502,unitsres,2160,361,2,Incorrect increse in number of units taking value from DOB CO,22v3
+1015287501,bldgarea,20734188,360570,2,Incorrect increse in building area taking value from 22v2,22v3
+4049727505,bldgarea,11782179,466988,2,Incorrect increse in building area taking value from 22v2,22v3
```

Comment:
I don't see this correction being applied. Below is what I see for bldgarea for this bbl 
<img width="355" alt="Screenshot 2022-11-03 at 12 18 50 PM" src="https://user-images.githubusercontent.com/41444592/199775812-e829f617-9c77-499b-a51d-fd6ce8ea2a6c.png">

Reply:
hopefully will be resolved when we build it on main